### ISSUE TITLE
When detaching from scratch session the d key is passed to open program.

### ISSUE THREAD

**Issue by Msouza91** [omerxx/tmux-floax#1]
Stumbled upon this while testing floax, if I have lazygit open when detaching it sent the d keystroke to lazygit which would trigger unwanted behaviour in a rebase specifically.

https://github.com/omerxx/tmux-floax/assets/19522556/156b5089-620f-4a4c-83a8-0dd6614da16d

**Comment by omerxx** [comment]
@Msouza91 I must say I can't reproduce, tried the exact same flow and I don't see anything sent.
Can you share your configuration for the plugin (if any defaults were changed)?
Trying to understand where this is coming from. When an existing popup is trigered it's doing two things
1. [setting bindings](https://github.com/omerxx/tmux-floax/blob/ecc0507a792a9f55529952c806e849c11093a168/scripts/utils.sh#L25)
2. [popping the window](https://github.com/omerxx/tmux-floax/blob/ecc0507a792a9f55529952c806e849c11093a168/scripts/utils.sh#L45)
There's nothing here that can suggest a keystroke. 

I'll try to search deeper and see whether I can reproduce

**Comment by Msouza91** [comment]
I switched the binding to open floax to u and my tmux prefix is ^-a. How can I capture more info to aid troubleshooting? I had keycastr on, it is showing what I did while it happens and it does happen 100% of the time

**Comment by omerxx** [comment]
@Msouza91 Yep, I think I know why. Floax has a feature that changes the paths in a rather naive way by `cd`ing into the path if it's different.
To disable is you can `set -g @floax-change-path 'false'` but I need to think about this further since it does send key to the window...

Happy to hear your thoughts as well.

**Comment by Msouza91** [comment]
Hey @omerxx sorry for the delayed response, got caught up with some stuff and forgot to come back.
The workaround indeed works, I haven't looked into the code yet, I'll schedule some time to have a look on the weekend hopefully, but I'm no wizard haha!

That change path thing might be a tricky one, pushing or popping from the directory jump history might be complicated to keep track of and still will be sending a command through the shell which could have other shortcut clash.
Maybe if could check if there is a process on the foreground then avoid changing directory until the process is terminated or sent to the background...

**Comment by omerxx** [comment]
@Msouza91 Interesting. I'm totally open for changes, not sure about the UX of it at all. I do know that 9/10 times when I pop the floating pane I'm annoyed by the fact it's not going to the directory I'm currently at (usually where neovim is open), that's why I added it
